crates/turborepo-lockfiles/src/npm.rs

```diff
@@ -5,22 +5,24 @@ use serde_json::Value;
 
 use super::{Error, Lockfile, Package};
 
+type Map<K, V> = std::collections::BTreeMap<K, V>;
```

Comment:
You might want to consider IndexMap here. It's a hash map that keeps insertion order. That's more performant than sorting. Doesn't matter in this case, but just in general.